app/templates/components/supplementary-file-browser.hbs

```diff
@@ -0,0 +1,26 @@
+{{#file-renderer download=selectedFile.links.download
+    width="100%" height="700"}}
+{{/file-renderer}}
+
+{{#if supplementList}}
```

Comment:
Following from another comment @abought made, I'm wondering if it makes sense to separate the file browser here as a component.  It will not be reused within our app.